go/libkb/notify_router.go

```diff
@@ -110,6 +110,8 @@ type NotifyListener interface {
 	SaltpackOperationProgress(opType keybase1.SaltpackOperationType, filename string, bytesComplete, bytesTotal int64)
 	SaltpackOperationDone(opType keybase1.SaltpackOperationType, filename string)
 	UpdateInviteCounts(keybase1.InviteCounts)
+	TeamTreeMembershipsPartial(keybase1.TeamTreeMembership)
```

Comment:
do we plan to have multiple team trees being loaded at the same time?

Reply:
Yeah, i have a plan to key it with sessionid in the next pr